### ISSUE TITLE
Training metric depth problem

### ISSUE THREAD

**Issue by a1400214194** [LiheYoung/Depth-Anything#210]
@LiheYoung  hellp, when I want to run train_momo.py, I met some questions
like Missing key(s) in state_dict: "core.core.pretrained.cls_token", "core.core.pretrained.pos_embed", "core.core.pretrained.mask_token", "core.core.pretrained.patch_embed.proj.weight", "core.core.pretrained.patch_embed.proj.bias", "core.core.pretrained.blocks.0.norm1.weight", "core.core.pretrained.blocks.0.norm1.bias"...
Unexpected key(s) in state_dict: "pretrained.cls_token", "pretrained.pos_embed", "pretrained.mask_token", "pretrained.patch_embed.proj.weight", "pretrained.patch_embed.proj.bias", "pretrained.blocks.0.norm1.weight", "pretrained.blocks.0.norm1.bias", "pretrained.blocks.0.attn.qkv.weight", "pretrained.blocks.0.attn.qkv.bias", "pretrained.blocks.0.attn.proj.weight", "pretrained.blocks.0.attn.proj.bias", "pretrained.blocks.0.ls1.gamma", "pretrained.blocks.0.norm2.weight", "pretrained.blocks.0.norm2.bias"...
It comes like "can't load the pretrain files and models ",how to do it?

**Comment by theoOmathimatikos** [comment]
There is probably a mismatch between the expected model architecture and the given one.
